http/gl3.xml.bak

```diff
@@ -0,0 +1,1887 @@
+<?xml version="1.0"?>
```

Comment:
Dammit, I keep accidentally committing things. In oC I committed my backup config files, here I commit my gl3.xml.bak, I even accidentally pushed directly to *both* of the stable branches of oC (which is strictly forbidden - everything is a PR!). And all today! :gah: